data/json/items/gun/shot.json

```diff
@@ -211,7 +236,11 @@
     "durability": 7,
     "barrel_length": 3,
     "magazine_well": 1,
-    "magazines": [ [ "shot", [ "saiga10mag", "saiga30mag" ] ] ],
+    "magazines": [
+      [ "shot", [ "saiga10mag", "saiga30mag" ] ],
+      [ "308", [ "falmag", "falbigmag" ] ],
```

Comment:
No ammo conversion kit exists for this (and doesn't need adding). As such those entries would never be used